recipes/computer_vision/object_detection/model_server/Containerfile

```diff
@@ -1,8 +1,9 @@
 FROM registry.access.redhat.com/ubi9/python-311:1-52
+ARG PORT
```

Comment:
This needs a default,

Reply:
I can change this but all the work for object_detection will live in #236